tests/conftest.py

```diff
@@ -640,18 +646,6 @@ def fixture_flow_cell_full_name() -> str:
     return "201203_A00689_0200_AHVKJCDRXX"
 
 
-@pytest.fixture(name="flow_cell_name")
-def fixture_flow_cell_name() -> str:
-    """Return flow cell name."""
-    return "HVKJCDRXX"
```

Comment:
Any reason why this has to be moved into `tests/apps/demultiplex/conftest.py` ? I argue it can lead to code duplication if its that "deep" in the fixture tree

Reply:
This fixture is used a bit strangely and seem to be dependent on other states. I do not want to refactor this also as I have gone a bit over the top already. So I just moved and made it more specific - connecting it to cgstats. It will have to be fixed in a future PR